ili/utils/ndes_pt.py

```diff
@@ -262,41 +268,52 @@ def load_nde_lampe(
             Defaults to True.
         theta_normalize (bool, optional): whether to z-normalize theta.
             Defaults to True.
+        engine (str, optional): dummy argument to match sbi interface.
+            Must be set to 'NPE' or will be overwritten.
         **model_args: additional arguments to pass to the model.
     """
+    if 'NPE' not in engine:
+        logging.warning(f'Engine {engine} not supported in lampe backend. '
```

Comment:
I think raising an error is probably more appropriate rather than guessing what the user wants to do. The user probably is more likely to know that they want to do NPE/NLE/NRE than knowing the specifics of each backend, so we should tell the user that the configuration they specify is invalid and they should make one of two changes. Please add this error check to the unit tests.